pkgs/common-updater/scripts/update-source-version

```diff
@@ -188,7 +188,7 @@ tempHash=$(printf '%0*d' "$hashLength" 0)
 if [[ -n "$sri" ]]; then
     # SRI hashes only support base64
     # SRI hashes need to declare the hash type as part of the hash
-    tempHash="$(nix to-sri --type "$oldHashAlgo" "$tempHash")"
+    tempHash="$(nix hash to-sri --type "$oldHashAlgo" "$tempHash")"
```

Comment:
The issue is that stable Nix does not support `hash` command so we need to detect it and use `to-sri` there.